lib/generators/doorkeeper/templates/migration.rb.erb

```diff
@@ -10,9 +10,9 @@ class CreateDoorkeeperTables < ActiveRecord::Migration<%= migration_version %>
       t.timestamps             null: false
     end
 
-    add_index :oauth_applications, :uid, unique: true
+    add_index :oauth_applications, :uid, unique: true, <%= primary_id_option %>
 
-    create_table :oauth_access_grants do |t|
+    create_table :oauth_access_grants, <%= primary_id_option %> do |t|
```

Comment:
I think `,` must be included in the template only if primary ID option is not null...

Reply:
included